src/App.jsx

```diff
@@ -14,33 +15,77 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [sortfiled, setSortfiled] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const visibileGoods = [...goodsFromServer].sort((a, b) => {
+    if (sortfiled === SORT_BY_ALPHABET) {
```

Comment:
Ensure that the corrected variable name `sortField` is used here instead of `sortfiled`.